extensions/wikia/TemplateClassification/TemplateClassification.hooks.php

```diff
@@ -105,17 +103,17 @@ public function onEditPageShowEditFormFields( \EditPage $editPage, \OutputPage $
 		if ( ( new Permissions() )->shouldDisplayEntryPoint( $context->getUser(), $title ) ) {
 			$types = $this->getTemplateTypeForEdit( $editPage->getTitle(), $wgCityId );
 
-			$out->addHTML( \Html::hidden( 'templateClassificationTypeCurrent', $types['current'],
-				['autocomplete' => 'off'] ) );
-			$out->addHTML( \Html::hidden( 'templateClassificationTypeNew', $types['new'],
-				['autocomplete' => 'off'] ) );
+			$out->addHTML( \Html::hidden( 'templateClassificationTypeCurrent', $types[ 'current' ],
+				[ 'autocomplete' => 'off' ] ) );
+			$out->addHTML( \Html::hidden( 'templateClassificationTypeNew', $types[ 'new' ],
+				[ 'autocomplete' => 'off' ] ) );
 
 			// add additional class to body for new templates in order to hide editor while template classification
 			// modal is visible and builder is available
 			if (
 				$wgEnablePortableInfoboxBuilderExt
 				&& $title->getArticleID() === 0
-				&& empty( $types['current'] && $types['new'] )
+				&& empty( $types[ 'current' ] ) && empty( $types[ 'new' ] )
```

Comment:
can we move `&& empty( $types[ 'new' ] )` to a separate line and move this logic to a separate function?